### DISCUSSION TITLE
JSON not available in Power Fx on Power Platform CLI

### DISCUSSION THREAD

**Discussion by rajyraman** [microsoft/Power-Fx#2734] [Ideas]
I am in the process of writing Power Platform extension for Bicep, and one of the pain point at the moment for me in the inability to use JSON function when I run pac power-fx run. When I try to use JSON it says it is not supported.

![image](https://github.com/user-attachments/assets/0bc68c2e-7b49-4c29-9fbe-2589a6fde65e)

I am unable to use the response properly because it is all mangled with text response in Bicep. If it is JSON I can at east parse it and chain it.

![image](https://github.com/user-attachments/assets/159e81ee-4e2b-483d-a9b1-646cfea7a9c4)

Please make JSON function available on Power Fx REPL on Power Platform CLI.

cc @MikeStall

**Comment by MikeStall** [comment]
Looks like we're misisng a call to https://github.com/microsoft/Power-Fx/blob/f891601ae69752a2907980c2ad2e51b552e3351b/src/libraries/Microsoft.PowerFx.Json/Environment/PowerFxConfigExtensions.cs#L16 

Will check...

**Comment by MikeStall** [comment]
this is fixed and will be available in next CLI version

**Reply by rajyraman** [reply]
@MikeStall - Checked with pac v1.37.4 and JSON now works. Thanks to you and the amazing Power Fx team for quickly fixing this.

**Comment by rajyraman** [comment]
Amazing!! Thank you Mike.
